tests/Functional/Fixtures/web/symfony.php

```diff
@@ -21,4 +21,4 @@
 $request = Request::createFromGlobals();
 $response = $kernel->handle($request);
 $response->send();
-$kernel->terminate($request, $response);
+//$kernel->terminate($request, $response);
```

Comment:
i think this is the only problem. i commented this line as i did not manage to get hhvm to log anything on travis. locally, it logged what i pasted in my comment. and i really can't explain why that could ever happen. i checked the codebase, nothing is assigning to $this->dispatcher, also not the extending classes...